modules/ezcontent_node/modules/ezcontent_article/config/update/ezcontent_article_update_8002.yml

```diff
@@ -0,0 +1,9 @@
+core.entity_form_display.node.article.default:
```

Comment:
This config should be part of user.role.editor.yml file not core.entity_form_display.node.article.default, please see.